src/redux/middleware/AnalyticsMiddleware.js

```diff
@@ -1,21 +1,36 @@
 import * as types from '../constants/actionTypes';
-
 // those action types that are listed here will be tracked
 const whiteList = [];
 
 export default class AnalyticsMiddleware {
 
-    constructor (args = {}) {
+    constructor(args = {}) {
         this.tracking = args.tracking;
```

Comment:
kanskje greit å throw hvis args ikke har tracking? Så får vi ikke feil down the line når man prøver å bruker tracking senere